src/Services/AppInstallActivationHandler.cs

```diff
@@ -51,63 +63,107 @@ protected override bool CanHandleInternal(ProtocolActivatedEventArgs args)
 
     protected async override Task HandleInternalAsync(ProtocolActivatedEventArgs args)
     {
-        await AppActivationFlowAsync(args.Uri.Query);
-    }
+        var uri = args.Uri;
+        var parameters = HttpUtility.ParseQueryString(uri.Query);
 
-    private async Task AppActivationFlowAsync(string query)
-    {
-        try
+        if (parameters != null)
         {
-            // Don't interrupt the user if the machine configuration is in progress
-            if (_setupFlowOrchestrator.IsMachineConfigurationInProgress)
+            var searchQuery = parameters.Get("search");
+            var wingetIdsQuery = parameters.Get("wingetIds");
+
+            if (!string.IsNullOrEmpty(searchQuery))
```

Comment:
Can you use `ActivationQueryType.Search.ToString()` instead of hard coding the string?

Reply:
Looks like the original code reference for this comment was lost, resolving but feel free to re-open if this is still applicable.

Reply:
Sorry.  The comment was on the wrong line.

I'm referring to `var searchQuery = parameters.Get("search");`.  The hardcoded "search" can be replaced with `ActivationQueryType.Search.ToString()` to get `parameters.Get(ActivationQueryType.Search.ToString())`